modules/examples/ExampleUser/example_user/example_userImpl.hpp

```diff
@@ -47,6 +49,7 @@ class example_userImpl : public example_userImplBase {
 
     // ev@3370e4dd-95f4-47a9-aaec-ea76f34a66c9:v1
     // insert your private definitions here
+    boost::uuids::uuid received_error;
```

Comment:
I think we should define a new type here, Everest::ErrorHandle or similar instead of using boost::uuid directly